wheel/pep425tags.py

```diff
@@ -139,17 +139,15 @@ def calculate_macosx_platform_tag(archive_root, platform_tag):
         for filename in filenames:
             if filename.endswith('.dylib') or filename.endswith('.so'):
                 lib_path = os.path.join(dirpath, filename)
-                versions_dict[lib_path] = extract_macosx_min_system_version(lib_path)
+                min_ver = extract_macosx_min_system_version(lib_path)
```

Comment:
I think the last field of the version tuple is irrelevant to the compatibility question? If so, should it be moved up here i.e. `min_ver = min_ver[0:2]`

Reply:
It need changes in more places. I push this changes in separate commit.

Reply:
👍